pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -522,3 +524,49 @@ def test_loc_with_mi_indexer():
         columns=["author", "price"],
     )
     tm.assert_frame_equal(result, expected)
+
+
+def test_getitem_str_slice():
+    # GH#15928
+    path = os.path.join(
```

Comment:
instead of doing this can you use datapath fixture, maybe
`datapath("reshape", "merge", "data", "quotes2.csv")`

Reply:
sure, just pushed with update